### PR TITLE
Mobile portfolio Pull 1

### PR DESCRIPTION
**Pull request by Danfall369**
My first pull request for this project.

- Add the `Home` page
- Implement `Html`
- Implement `CSS`
- Add styling provided by `Microverse`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      <p id="welcome">Welcome</p>
+      <nav>
+        <img id="menu" class="menu-icon" src="Style/icons/menu.svg" alt="menu-icon">
+      </nav>
+    </div>
+  </header>
+
+  <main class="headline">
+    <p class="primary">Hey there. I’m Mariane<br>
+      <span class="secondary">I’m a software developer</span>
```
Comment (mukhammadrizooff) [inline index.html:30]: - [ ] [REQUIRED] Good job so far! The `main title` (e.g., "I'm Marianne ...") should be a `h1` tag. If you want that google knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info 🙏🏼

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      and
+      experience! If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.</p>
+    <div id="social-cont">
+      <ul id="social-media-icons">
+        <li><img id="twitter" class="media-icon" src="Style/Social Media Icons/twitter.svg" alt="twitter"></li>
+        <li><img id="linkedin" class="media-icon" src="Style/Social Media Icons/Linkedin icon.svg" alt="Linkedin"></li>
+        <li><img id="medium" class="media-icon" src="Style/Social Media Icons/medium.svg" alt="medium"></li>
+        <li><img id="github" class="media-icon" src="Style/Social Media Icons/github.svg" alt="github"></li>
+        <li><img id="angellist" class="media-icon" src="Style/Social Media Icons/angellist.svg" alt="angellist"></li>
```
Comment (mukhammadrizooff) [inline index.html:42]: - [ ] [REQUIRED] Good job so far! The list of icons uses a `ul`✔️ tag to group them, and `li`✔️  per icon; each icon has a `link`(`a` tag)❌. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly implement my suggestion please 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Danfall369, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] It would be nice if you add more informations about your project to your `Pll Request` summary 👍🏼 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Danfall369) [reply index.html:30]: @mukhammadrizooff done with this

File: index.html (hunk 2 above)
Comment (Danfall369) [reply index.html:42]: @mukhammadrizooff also done with this

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Danfall369, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive `PR` summary ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**